CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.1.1] - 2018-10-08
+### Added
+   - Shruggies! (#15, @wadadli)
+   - CircleCI support for automating tests (#8, @wadadli)
+   - Added a changelog
+
+### Changed
+
+### Fixed
```

Comment:
WOW this is super duper awesome, I'm really digging the change log adoption, and all of the contributors!